src/ocean/task/util/Timer.d

```diff
@@ -83,7 +80,27 @@ unittest
 {
     initScheduler(SchedulerConfiguration.init);
 
-    .timer = new typeof(timer);
+    class SimpleTask : Task
+    {
+        override public void run ( )
+        {
+            for (int i = 0; i < 10; ++i)
+                .wait(10);
+        }
+    }
+
+    auto task = new SimpleTask;
+    theScheduler.schedule(task);
+    theScheduler.eventLoop();
+}
+
+unittest
+{
+    // same as previous, but test allocated slot count
+
+    initScheduler(SchedulerConfiguration.init);
+
+    .timer = new typeof(.timer);
```

Comment:
And is it necessary here?

Reply:
Yes, for a reason explained above (it tests exact allocated slot count)

Reply:
Hmm, I don't understand that comment. What's a "slot"? Why is it necessary to re-allocate the timer? Doc enhancement please :)

Reply:
https://github.com/sociomantic-tsunami/ocean/pull/234/files#diff-a3ce7f2ae79a2097ba597486e0d9a30fR121 + comment 3 lines above

Which doc?

Reply:
Some documentation of what this unittest is testing. It's not clear to me.

>Why is it necessary to re-allocate the timer?
"Reallocate a fresh timer so the test result is not affected by previous unittests" ?

Reply:
Updated